server/routes/regionRoutes.js

```diff
@@ -14,10 +15,13 @@ router.get('/', handlerWrapper(regionHandlerGet));
 
 router.get('/count', handlerWrapper(regionHandlerGetCount));
 
-router.get('/:regionId', handlerWrapper(regionHandlerGetByRegionId));
+router.get('/:region_id', handlerWrapper(regionHandlerGetByRegionId));
 
-router.post('/', handlerWrapper(regionHandlerPost));
+router.get(
+  '/shape/:region_id',
+  handlerWrapper(regionHandlerGetShapeByRegionId),
```

Comment:
I think for this instead we should allow for a link that ends in .geojson, and it should be part of the /region path.   /region/:region_id.geojson  - is that possible?  or do we need a different prefix, like /region/shape ?

Reply:
That route returns the geojson for download and it is also a /region endpoint @ZavenArra 
![image](https://user-images.githubusercontent.com/28721558/158075771-1f4e8e0a-bf0e-47ba-8eff-67a4b61a0d51.png)